qemu/tests/cfg/virtio_trace_pipenb.cfg

```diff
@@ -0,0 +1,11 @@
+- virtio_trace_pipenb:
+    only Linux
+    type = virtio_trace_pipenb
+    serials += " vs2"
+    serial_type_vs2 = virtserialport
+    chardev_backend_vs2 = pipe
+    variants:
+        - @default:
+            machine_type = 'pc'
+        - q35:
+            machine_type = 'q35'
```

Comment:
No need to add this `variants`, we will distinguish pc and q35 by argument "--machines"